browser/src/components/NodeSummary.tsx

```diff
@@ -87,6 +93,24 @@ function createdComponent(graphManager: GraphManager, componentName: string, nod
   return ""
 }
 
+export function NodeSearch({ graphManager, weightService, nodeName }: SearchProps) {
+  //search for the top five choices the nodeName could be in the Dagger-Graph based on the nodeName
+  var searchResult =  graphManager.getMatches( "", nodeName.trim().toLowerCase(), 5, false);
```

Comment:
Let's start from return top 1 result, since in this PR we always use the first match result